lsp-clients.el

```diff
@@ -511,6 +511,25 @@ responsiveness at the cost of possibile stability issues."
                   :add-on? t
                   :server-id 'angular-ls))
 
+
+;; TeX
+(defgroup lsp-tex nil
+  "LSP support for TeX and friends, using Digestif."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/astoff/digestif/"))
+
+(defcustom lsp-clients-digestif-executable "digestif"
+  "Command to start the Digestif language server."
+  :group 'lsp-tex
+  :risky t
+  :type 'file)
+
+(lsp-register-client
+ (make-lsp-client :new-connection (lsp-stdio-connection lsp-clients-digestif-executable)
```

Comment:
for future reference it is a bit better to wrap `lsp-clients-digestif-executable` in lambda otherwise the changes to the variable won't take effect unless lsp-clients.el is evaluated again.